src/open_samus_returns_rando/files/levels/s036_area3c.lua

```diff
@@ -322,13 +322,7 @@ function s036_area3c.OnGamma_007_B_Generated(_ARG_0_, _ARG_1_)
   end
 end
 function s036_area3c.OnEnter_Gamma_007_Dead()
-  if Scenario.ReadFromBlackboard("Arena_Gamma_007_AllDead", false) then
-    Game.SetSubAreaCurrentSetup("collision_camera_013", "PostGamma_007", true)
-    Game.SetSubAreaCurrentSetup("collision_camera_027", "PostGamma_007", true)
-    if Game.GetEntity("SpawnGroup008") ~= nil then
-      Game.GetEntity("SpawnGroup008").SPAWNGROUP:EnableSpawnGroup()
-    end
-  end
+  s036_area3c.OnSubAreaChange()
```

Comment:
What's that supposed to do? It's missing every parameter for this function.

Reply:
It's supposed to call that function, but maybe it's breaking because of the missing parameters. Or maybe it just won't work like this at all (though why does this not crash console but it does crash citra)

Reply:
I was aware of what it is doing technically 😛. It was more about what this logically does.

Reply:
Oh. The goal is to only change the Gamma_007 ccs after entering the transport cc. For some reason if enter the side room cc first then leave the Gamma section the game will crash if you enter the right cc past the transport. I didn't test leaving the scenario or taking the upper tunnel but yeah it breaks otherwise on console. This change makes it work on console, but crashes on citra, which is why I'm frustrated and confused